companies/hagengrote.json

```diff
@@ -0,0 +1,20 @@
+{
+    "slug": "hagengrote",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "commerce"
+    ],
+    "name": "Hagen Grote GmbH",
+    "address": "Grüner Dyk 36\n47803 Krefeld\nDeutschland",
+    "phone": "+49 2151 607090",
+    "fax": "+49 2151 6070999",
+    "email": "datenschutz@hagengrote.de",
```

Comment:
As this email is specific for privacy-related requests, we can set the `suggested-transport-medium` to `email`.